src/globus_sdk/experimental/auth_requirements_error/_variants.py

```diff
@@ -305,5 +194,15 @@ def to_auth_requirements_error(self) -> GlobusAuthRequirementsError:
         return GlobusAuthRequirementsError(
             authorization_parameters=authorization_parameters,
             code=self.code,
-            extra=self.extra_fields,
+            extra=self.extra,
+        )
+
+
+def _validate_consent_required_literal(
+    name: str, value: t.Any
+) -> Literal["ConsentRequired"]:
+    if not isinstance(value, str) or value != "ConsentRequired":
+        raise _validators.ValidationError(
+            f"'{name}' must be the string 'ConsentRequired'"
         )
+    return t.cast(Literal["ConsentRequired"], value)
```

Comment:
Could this simply be:

```suggestion
    return "ConsentRequired"
```

Reply:
:facepalm: 
I would be surprised if this didn't work. Will confirm momentarily.